app/views/comments/_comment.html.haml

```diff
@@ -6,7 +6,14 @@
   .container
     %p= "#{comment.user.first_name} #{comment.user.last_name} @#{comment.user.nickname}"
     %p= comment.content
-    %i.bi.bi-chat-right.toggle-commentscomment-form{id: "toggle-commentcomment-form-#{comment.id}", data: { comment_id: comment.id }}
+    .container.d-flex.comment-icons-container
+      .like-unlike-container{id: "like_unlike_container-#{comment.id}"}
+        - if current_user.liked_comments?(comment)
+          = render partial: 'partials/unlike', locals: { likable: comment, current_user: current_user}
+        - else
+          = render partial: 'partials/like', locals: { likable: comment }
+      = render partial: 'partials/like-stats', locals: { likable: comment }
```

Comment:
This has some partials potencial 😄 

So you can reuse the logic in the `_show-comment.html.haml` file